qdrant-landing/content/articles/what-is-quantization.md

```diff
@@ -0,0 +1,249 @@
+}
+```
+
+Product Quantization is generally more memory efficient than **Scalar Quantization**, being able to reduce the memory footprint by up to **64x** while still retaining much of the essential information. However, the computational overhead of mapping sub-vectors to centroids results in the **slowest performance** compared to the other methods.
+
+If your application requires high precision or real-time performance, this slower computation might be a significant factor to consider.
+
+# 3. What is Binary Quantization?
+
+![](/articles_data/what-is-vector-quantization/astronaut-white-surreal.jpg)
+
+[Binary Quantization](https://qdrant.tech/documentation/guides/quantization/#binary-quantization) is an excellent option if you're looking to **reduce memory** usage while also achieving a significant **boost in speed**. It works by converting high-dimensional vectors into simple binary (0 or 1) representations.
+
+* Values greater than zero are converted to 1 
+* Values less than or equal to zero are converted to 0
+
+Let's take our initial example of a 1536-dimensional vector that requires **6KB** of memory, 4 bytes for each float32 value.
+
+After Binary Quantization, each dimension is reduced to 1 bit (1/8 byte), so the memory required is: 
+
+$$
+\frac{1536 \text{ dimensions}}{8 \text{ bits per byte}} = 192 \text{ bytes}
+$$
+
+
+This leads to a **32x** memory saving.
+
+
+![Binary Quantization example](/articles_data/what-is-vector-quantization/binary-quantization.png)
+
+
+Qdrant automates the Binary Quantization process during indexing. As vectors are added to your collection, each 32-bit floating-point component is converted into a binary value according to the configuration you define. 
+
+Here’s how you can set it up:
+
+```python 
+collection_config = {
+    "vectors": {
+        "size": 1536,
+        "distance": "Cosine"
+    },
+    "quantization_config": {
+        "binary": {
+            "always_ram": True
+        }
+    }
+}
+```
+
+Binary Quantization is by far the quantization method that will give you the most processing **speed gains** when compared to Scalar and Product Quantizations. This is because the binary representation allows the system to use highly optimized CPU instructions, such as [XOR](https://en.wikipedia.org/wiki/XOR_gate#:~:text=XOR%20represents%20the%20inequality%20function,the%20other%20but%20not%20both%22.) and [Popcount](https://en.wikipedia.org/wiki/Hamming_weight), for fast distance computations.
+
+It can speed up search operations by **up to 40x**, depending on the dataset and hardware. Here's the speed comparison of all three methods:
+
+<img src="/articles_data/what-is-vector-quantization/speed-methods.png" alt="Speed by Quantization Method" width="700">
+
+
+But it’s important to note that not all models are equally compatible with Binary Quantization. Some models may suffer a larger loss in accuracy when quantized, so it's recommended to use it on models of **at least 1024 dimensions.** 
```

Comment:
As I remember, cohere 768-embeddings work fine with BQ. I propose to ask someone from ML team if we can decrease recommended dimensions to 768 or even 512

Reply:
I've added a Disk & RAM Storage section!